### PR TITLE
Add - to support dash character in JSitor liquid tag

### LINKED ISSUE #4823: JSitor liquid tag does not support id with '-' character
**Describe the bug**
This `{% jsitor 2o-syYxmi %}` is a valid id `https://jsitor.com/2o-syYxmi` and should embed the content. However while adding that in blog, it throws the below error -
image

**To Reproduce**
Try `{% jsitor 2o-syYxmi %}`

**Expected behavior**

It should allow id with '-'

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Browser:
- Version:

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

### PR DESCRIPTION
**Pull request by helmihidzir**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
This is to support JSitor id with '-' character.

## Related Tickets & Documents

Closes #4823

### CODE REVIEW

**Comment by ashvin777** [comment]
The coverage jobs are stuck. @helmihidzir could you please check.

**Comment by helmihidzir** [comment]
@ashvin777 Yes I will look into it. @rhymes Thank you for the review, I will add the URL and the single ID option test.

File: app/liquid_tags/jsitor_tag.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class JsitorTag < LiquidTagBase
   PARTIAL = "liquids/jsitor".freeze
   URL_REGEXP = /\A(https|http):\/\/jsitor\.com\/embed\/\w+[?a-zA-Z&]*\Z/.freeze
```
Comment (rhymes) [inline app/liquid_tags/jsitor_tag.rb]: Tihs doesn't work with and ID with the dash, like `https://jsitor.com/embed/2o-syYxmi`. Don't forget to add a test for it!

Comment (helmihidzir) [reply app/liquid_tags/jsitor_tag.rb]: should have read last pr #4762 and understand the code more. sorry about that. gonna add it now

**Comment by helmihidzir** [comment]
Thank you @rhymes for the guidance and give me the opportunity to work on this one. 😄

**Pull request merged**